manifest.toml

```diff
@@ -1,103 +0,0 @@
-packaging_format = 2
```

Comment:
(NB: the point is that we're ideally supposed to use packaging v2 now, not the old v1 .. though it still works :grimacing: )

Reply:
Yes, I have this in mind...
But it will take a bit of time for me to dig in and learn the new format 😅

Reply:
Also, I don't know anything about Nim, so I took the path of copy pasting Nitter package (v1) and adapt it for Buckets (and I have to say that it worked surprisingly well. Never got a nearly working package so fast)